EventFilter/CTPPSRawToDigi/plugins/CTPPSPixelDigiToRaw.cc

```diff
@@ -0,0 +1,198 @@
+// -*- C++ -*-
+//
+// Package:    CTPPSPixelDigiToRaw
```

Comment:
We have removed "CT" from the name of the detector: couldn't it then avoid using it in the class names and variables, at least in the new code? (Then at some point also the old code could be reworded accordingly)

Reply:
This part is still to be done (sorry). We didn't use CTPPS for the new, top level path, but as there were already some code in the release with this name, we need more time to check the consequences (see forthommel comment below). Can we go like this?